content/en/blog/2024/otel-arrow-production/index.md

```diff
@@ -2,10 +2,9 @@
 title: OpenTelemetry Protocol with Apache Arrow in Production
 linkTitle: OpenTelemetry Arrow in Production
 date: 2024-09-25
-author:
-  >- # If you have only one author, then add the single name on this line in quotes.
-  [Joshua MacDonald](https://github.com/jmacd) (ServiceNow, Inc), [Laurent
-  Querel](https://github.com/lquerel) (F5, Inc)
+author: >-
+  [Joshua MacDonald](https://github.com/jmacd) (ServiceNow), [Laurent
+  Querel](https://github.com/lquerel) (F5)
```

Comment:
@jmacd - we don't typically include "Inc" like qualifiers in blog post credentials, neither did you in your [2023 post](https://opentelemetry.io/blog/2023/otel-arrow/), so I stripped them (FYI).